packages/notifications/src/Collection.php

```diff
@@ -5,10 +5,13 @@
 use Illuminate\Support\Collection as BaseCollection;
 use Livewire\Wireable;
 
+/**
+ * @extends BaseCollection<array-key, Notification>
+ */
 class Collection extends BaseCollection implements Wireable
 {
     /**
-     * @param  array<array<string, mixed>>  $items
+     * @param  array<array-key, Notification>  $items
```

Comment:
I try and avoid `array<array-key` as it usually feels purposeless, should I think about this differently for some reason? Happy to learn

Reply:
It's shorthand for `int | string`. Though this looks like might be error. Will confirm and check